packages/jest-circus/src/eventHandler.ts

```diff
@@ -195,6 +196,10 @@ const eventHandler: Circus.EventHandler = (
       event.test.invocations += 1;
       break;
     }
+    case 'test_fn_start': {
```

Comment:
hmm, maybe we want it in the `test_start` event instead?

Reply:
`test_start` is dispatched here: 
https://github.com/facebook/jest/blob/ca47512c77e6a325bca734a35a4ad15a45dee8aa/packages/jest-circus/src/run.ts#L93-L97 and `test_fn_start` is dispatched here: https://github.com/facebook/jest/blob/ca47512c77e6a325bca734a35a4ad15a45dee8aa/packages/jest-circus/src/run.ts#L165-L169

Should I move it to `test_start`?

Reply:
let's keep it in `test_fn_start`